package.json

```diff
@@ -27,13 +27,13 @@
     "rdf-parser-microdata": "^0.3.0",
     "rdf-parser-n3": "^0.3.0",
     "rdf-parser-rdfxml": "^0.3.0",
+    "rdf-ext": "^0.3.0",
```

Comment:
why do you want to add this as dependency? shouldn't this be.. injected?
Actually for browserify purposes one could just `--ignore rdf-ext` maybe, so it should be fine.